### ISSUE TITLE
Missing 1.17.8 version of @graphql-codegen/typescript-apollo-angular

### ISSUE THREAD

**Issue by UlisesAC4** [dotansimha/graphql-code-generator#4779]
**Describe the bug**
 

After initializing with the cli tool, cannot install new dependencies added to my package json.

**To Reproduce**

* Install `@graphql-codegen/cli`, at this moment the version that installs is "1.17.8"
* Initialize configuration with `npx graphql-codegen init`
* Choose angular as target and everything is left as is
* Run `npm install` with the new dependencies added automatically to my package.json which are:
  * "@graphql-codegen/typescript": "1.17.8",
  * "@graphql-codegen/typescript-operations": "1.17.8",
  * "@graphql-codegen/typescript-apollo-angular": "1.17.8",
  * "@graphql-codegen/introspection": "1.17.8"

<!--
#############
#############
🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 

PLEASE READ THE FOLLOWING: 
You can use this template to create a live sandbox of the issue: 
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template
Make sure sure fork this template, and run `yarn generate` in the terminal. 
Please make sure the Codegen and plugins version under `package.json` matches yours. 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ ISSUES WITH A LIVE REPRODUCTION WILL BE ANSWERED QUICKLY ~~~~~~~
~~~~~~ ISSUES WITHOUT REPRODUCTION MIGHT GET CLOSED WITHOUT RESPONSE ~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#############
#############
-->



1. My `codegen.yml` config file:

```yml
overwrite: true
schema: "http://localhost:4000"
documents: "src/**/*.graphql"
generates:
  src/generated/graphql.ts:
    plugins:
      - "typescript"
      - "typescript-operations"
      - "typescript-apollo-angular"
  ./graphql.schema.json:
    plugins:
      - "introspection"
```

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

**Environment:**

- OS: Archlinux
- `@graphql-codegen/cli@1.17.8`: 
- NodeJS: 12.18.3


**Additional context**
<!-- Add any other context about the problem here. -->

After seeing https://www.npmjs.com/package/@graphql-codegen/typescript-apollo-angular?activeTab=versions it is clear that there are only alpha versions of 1.17.8

I remember this working fine weeks ago, but I am not sure if these were the exact dependencies.

**Comment by dotansimha** [comment]
It's a bug with `init` command, I'll take a look, thanks @UlisesAC4 !

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/4809

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/cli@1.17.9`
